HW2/Fibonacci.java

```diff
@@ -0,0 +1,21 @@
+package HW2;
+public class Fibonacci {
+    public static void main(String args[]) {
+        int n = 100;
+        fibo(n);
```

Comment:
not working. try to run it in your idea

Reply:
Fibonacci.java updated, commit pushed

Reply:
in this task you should print all numbers from Fibonacci line, not only the last one

Reply:
change has been made for print all numbers